.github/workflows/ci.yml

```
@@ -27,19 +27,25 @@ jobs:
+
+      - uses: actions/setup-node@v4
         with:
-          node-version: '18.19.1'
+          node-version: 22
+
       - uses: oven-sh/setup-bun@v1
         with:
           bun-version: 1.1.10
+
       - run: bun install -y
         if: "${{ steps.cache-nodemodules.outputs.cache-hit != 'true' }}"
```

actionlint (1.7.7)

property "cache-nodemodules" is not defined in object type {}

(expression)